prompts/sources.go

```diff
@@ -148,13 +148,55 @@ func sourceBaseForm(quickstart *Quickstart) (*QuickstartState, error) {
 
 func AddToSource(name string, currentSource *workflow.Source) (*workflow.Source, error) {
 	addOpenAPIFile := false
+	selectedDoc := "new"
 	if _, err := tea.NewProgram(charm_internal.NewForm(huh.NewForm(
-		charm_internal.NewBranchPrompt("Would you like to add an openapi file to this source?", &addOpenAPIFile)),
-		fmt.Sprintf("Let's add to the source %s", name))).
+		huh.NewGroup(
+			huh.NewSelect[string]().
+				Title("Would you like to modify an existing OpenAPI document or add a new one?").
+				Options(huh.NewOptions(append([]string{"new"}, getCurrentSources(currentSource)...)...)...).
```

Comment:
We should be consistent with where the `new` option appears and what it looks like. In the previous menu where i'm selecting the source i want to configure, `new` is at the bottom (and is worded as `new source`). IMO both should be at the bottom, and both should say exactly `+ New <Thing>` where Thing is `Source` or `OpenAPI Document`

Reply:
The reason for the `+` is to visually distinguish it from the other options

Reply:
Will follow up with the + in a new PR

Reply:
https://linear.app/speakeasy/issue/SPE-2843/improve-edits-and-loopback-for-configure